### PR TITLE
Normalize units tests

### PR DESCRIPTION
**Pull request by cbron**

### CODE REVIEW

File: tests/integration/rbac_test.go [hunk 1]
```diff
@@ -102,7 +107,7 @@ func rbacTests(t *testing.T, when spec.G, it spec.S) {
 		it("rio-standard user should not be able to escalate privilege on global permissions", func() {
 			var testService testutil.TestService
 			testService.Kubeconfig = standardUser.Kubeconfig
-			err := testService.CreateExpectingError(t, "--global-permission", "list rio.cattle.io/services", "nginx")
+			err := testService.CreateExpectingError(t, "--global-permission", "\"list rio.cattle.io/services\"", "nginx")
```
Comment (StrongMonkey) [inline tests/integration/rbac_test.go]: why adding extra quotes? I think that's causing CI to fail.

File: cli/cmd/ps/ps_services.go [hunk 2]
```diff
@@ -114,7 +115,7 @@ func (p *Ps) services(ctx *clicontext.CLIContext) error {
 			continue
 		}
 		var gitwatcher *webhookv1.GitWatcher
-		gitwatchers, err := ctx.Gitwatcher.GitWatchers(namespace).List(metav1.ListOptions{})
+		gitwatchers, err := ctx.Gitwatcher.GitWatchers(service.(*riov1.Service).Namespace).List(metav1.ListOptions{})
```
[the remark this answers is not in the document]
Comment (StrongMonkey) [reply cli/cmd/ps/ps_services.go]: Yes, it is fine.

File: cli/pkg/tables/service.go [hunk 3]
```diff
@@ -26,7 +26,7 @@ func NewService(cfg Config) TableWriter {
 		{"SCALE", "{{.Obj | scale}}"},
 		{"WEIGHT", "{{.Obj | formatWeight}}"},
 		{"CREATED", "{{.Obj.CreationTimestamp | ago}}"},
-		{"DETAIL", "{{serviceDetail .Service .Pods .GitWatcher}}"},
+		//{"DETAIL", "{{serviceDetail .Service .Pods .GitWatcher}}"}, // todo: fix
```
[the remark this answers is not in the document]
Comment (StrongMonkey) [reply cli/pkg/tables/service.go]: `{"DETAIL", "{{serviceDetail .Data.Service .Data.Pods .Data.GitWatcher}}"},` @cbron try this

File: cli/cmd/weight/weight.go [hunk 4]
```diff
@@ -71,58 +81,44 @@ func scaleAndAllocate(ctx *clicontext.CLIContext, args []string, pause bool, inc
...
+	//		s.Spec.RolloutConfig = &riov1.RolloutConfig{
+	//			Pause:           pause,
+	//			Increment:       increment,
+	//			IntervalSeconds: interval,
+	//		}
+	//		return nil
+	//	})
+	//
+	//	errs = append(errs, err)
+	//}
```
Comment (StrongMonkey) [inline cli/cmd/weight/weight.go]: There are still a lot of comments. @cbron are you address those?

Comment (cbron) [reply cli/cmd/weight/weight.go]: Yea that will all be addressed in #682

**Pull request merged**